recipes-browser/packagegroups/packagegroup-wpewebkit-depends.bb

```diff
@@ -254,31 +264,26 @@ RDEPENDS:packagegroup-wpewebkit-depends-tests = " \
     curl \
     gdb \
     php \
-    ${@bb.utils.contains('BBFILE_COLLECTIONS', 'meta-python2', 'python python-subprocess32 python-pygobject python-psutil', '', d)} \
+    ${@bb.utils.contains('BBFILE_COLLECTIONS', 'meta-python2', 'python python-subprocess32 python-pygobject python-psutil', 'python-is-python3', d)} \
```

Comment:
Since python2 is not longer a requirement (https://docs.yoctoproject.org/migration-guides/migration-3.1.html). I'd suggest to remove the meta-python2 and any kind of reference to it.

We can do it in a separate MR.

Reply:
Added it https://github.com/Igalia/meta-webkit/pull/437/commits/2b2b841473b3b809a0f1b7478d7d7a7509fabc9c ( https://github.com/Igalia/meta-webkit/pull/437/commits )

Reply:
https://github.com/Igalia/meta-webkit/pull/437/commits merged.

Reply:
good riddance, i will remove references to meta-python2 as well